python_modules/dagster/dagster/core/definitions/assets_info.py

```diff
@@ -166,6 +166,14 @@ def from_graph(graph_def: GraphDefinition) -> "AssetsJobInfo":
             asset_deps=asset_deps,
         )
 
+    @property
+    def asset_info_by_node_input_handle(self) -> Mapping[NodeInputHandle, AssetInfo]:
```

Comment:
Prefer `asset_infos_by_node_input_handle`, unless you'd argue that "info" is an [uncountable noun](https://www.englishclub.com/grammar/nouns-countable-un.htm) and a list of AssetInfo objects should be called `asset_info`?

Reply:
yeah the word "infos" was kinda unaesthetic to me. i don't have strong feelings about it though so I can change it if you feel less ambivalent than I do